js/PulsarSortableComponent.js

```diff
@@ -0,0 +1,154 @@
+'use strict';
+
+/* jshint: global e, ui */
+
+var $ = require('jquery');
+
+function PulsarSortableComponent(html, window) {
+    this.$html = html;
+    this.window = window;
+}
+
+PulsarSortableComponent.prototype.init = function () {
+
```

Comment:
Can we make this consistent with the other function blocks? (I won't comment on all other occurrences, but there a couple more below) I vote for no line breaks on the first and last line of a function as per line 7.